src/Protocol/PortableContact.php

```diff
@@ -720,7 +720,7 @@ private static function parseNodeinfo1($nodeinfo_url)
 
 		$server['register_policy'] = Register::CLOSED;
 
-		if (is_bool($nodeinfo['openRegistrations']) && $nodeinfo['openRegistrations']) {
+		if (isset($nodeinfo['openRegistrations']) && is_bool($nodeinfo['openRegistrations']) && $nodeinfo['openRegistrations']) {
```

Comment:
The whole line could have been just `if (!empty($nodeinfo['openRegistrations'])) {`.

Reply:
You are right.